protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+
+When the `market creation metric` for a party is `>0` and the reward account balance for a specific reward asset is also `>0` (i.e. when a creator is rewarded):
+
+- A flag is added for each of the recorded `funders` of the reward account paired with the market and reward asset.
+This flag is used to prevent a recurring transfer from funding a creation reward in the same asset more than once.
+See the [transfers](./0057-TRAN-transfers.md) spec.
+- The list of funders for the reward account is cleared after the account is emptied (i.e. after one or more parties are rewarded for market creation).
+
+Market creation reward metrics (both each market's `cumulative volume` and the `payout record flags` to identify [market, payout asset, funder] combinations that have already been rewarded) are stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and will be restored after a checkpoint restart.
+
+
+
+## Reward accounts
+
+Trading reward accounts are defined by the reward asset (the asset in which the reward is paid out), the market, and the reward type (metric). 
+That is, there can be multiple rewards with the same type paid in different assets for the same market.
+
+Note that the market settlement asset has nothing to do in particular with the asset used to pay out a reward for a market. 
+That is, a participant might recieve rewards in the settlement asset of the market, in VEGA governance tokens, and in any number of other unrelated tokens (perhaps governance of "loyalty"/reward tokens issued by LPs or market creators, or stablecoins like DAI).
+
+Reward accounts can be individually funded by normal single transfers. They can also be funded on an ongoing (recurring) basis in groups, pro-rata by their relative total reward metrics for the given reward type. See [transfers](./0057-TRAN-transfers.md).
+
+Reward accounts and balances must be saved in [LNL checkpoints](./0073-LIMN-limited_network_life.md) to ensure all funds remain accounted for accross a restart.
 
-Reward accounts and balances are to be saved in [LNL checkpoint](./0073-LIMN-limited_network_life.md). 
 
 ## Reward distribution
 
 All rewards are paid out at the end of any epoch *after* [recurring transfers](0057-TRAN-Transfers.md) have been executed. 
-There are no fractional payouts and no delays.
+The entire reward account balance is paid out every epoch unless the total value of the metric over all parties is zero (there are no fractional payouts). 
+There are no payout delays, rewards are paid out instantly at epoch end.
 
-### For fee based metrics
+Rewards will be distributed pro-rata by the party's reward metric value to all parties that have metric values `>0`. That is, if we have reward account balance `R` and parties `p_1 – p_n` with non-zero metrics on the market in question:
```

Comment:
market proposer bonus is an exception as the total balance is divided equally between all the proposers of the markets that are eligible for it at the given point in time. Unless you're suggesting to do it pro-rata of the value generated? that's also possible.

Reply:
I tweaked the definition of the reward metric calculation above such that it is always either `0` or `1` (and added an explanation as to why) which has the effect of making the pro-rata distribute it equally but without having to write up two different bits of logic.

As long as if the same party created 2 (or more) markets that become eligible in the same epoch that party will receive 2 (or more) times the payout of a party with just one eligible market then we should be good.

Reply:
I've added a clarification here too.